HW5/app/src/main/java/com/kodeco/android/countryinfo/ui/components/CountryDetailsScreen.kt

```diff
@@ -1,15 +1,99 @@
 package com.kodeco.android.countryinfo.ui.components
 
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.Button
+import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.sp
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.navigation.NavController
+import coil.compose.AsyncImage
+import com.kodeco.android.countryinfo.types.Country
 
 // TODO fill in the CountryDetailsScreen. NOTE this is above-and-beyond the required
 //  section of the homework assignment.
 @Composable
-fun CountryDetailsScreen() { }
+fun CountryDetailsScreen(country: Country, viewModel: CountryDetailsViewModel) {
```

Comment:
Note you could also pass the `country` into the viewmodel that way the model. That way you aren't having to pass both here in the screen composable.